x-pack/plugins/ml/public/application/jobs/new_job/pages/index_or_search/preconfigured_job_redirect.ts

```diff
@@ -59,7 +63,20 @@ function getWizardUrlFromCloningJob(createdBy: string | undefined, job: Job, dat
       break;
   }
 
-  const indexPatternId = getIndexPatternIdFromName(datafeed.indices.join());
+  return `jobs/new_job/${page}?index=${dataViewId}&_g=()`;
+}
+
+async function getDataViewIdFromName(
+  datafeed: Datafeed,
+  dataViewsContract: DataViewsContract
+): Promise<string | null> {
+  if (dataViewsContract === null) {
+    throw Error('ARGGHH');
```

Comment:
Did you mean to leave `ARGHH` in!?

Reply:
10 points for finding the deliberate mistake that I definitely intended to leave in.

Reply:
Updated in https://github.com/elastic/kibana/pull/116455/commits/7ee9241872df9cecf890ae7dce916bfa3f491d84